stefaniaPopescu26/assignment2/TreeTest.java

```diff
@@ -0,0 +1,51 @@
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class TreeTest {
+
+    public Tree<Integer> getTestTree1() {
+        Tree<Integer> tree = new Tree<>();
+        tree.add(null, 0, 7);
+        tree.add(7, 0, 3);
+        tree.add(7, 1, 4);
+        tree.add(3, 0, 2);
+        tree.add(4, 1, 8);
+        tree.add(3, 1, 5);
+        tree.add(2, 0, 1);
+        tree.add(2, 1, 6);
+
+        return tree;
+    }
+
+    public Tree<Character> getTestTree2() {
+        Tree<Character> tree = new Tree<>();
+        tree.add(null, 0, 'L');
+        tree.add('L', 0, 'e');
+        tree.add('e', 0, 'H');
+        tree.add('e', 1, 'l');
+        tree.add('L', 1, 'o');
+        tree.add('o', 1, '!');
+
+        return tree;
+    }
+
+    @Test
+    public void testAdd1() {
```

Comment:
This test has all the test cases that the other test have too, there is no need for this repetition :)